contributing/code/bugs.rst

```diff
@@ -33,7 +33,9 @@ If your problem definitely looks like a bug, report it using the official bug
 * Give as much detail as possible about your environment (OS, PHP version,
   Symfony version, enabled extensions, ...);
 
-* If you want to provide a stack trace you got on an HTML page, be sure to
+* If there was an exception, please provide the :doc:`stack trace
```

Comment:
```suggestion
* If there was an exception and wanted to report it, it is valuabe to provide the :doc:`stack trace
```

Reply:
```diff
- If there was an exception and wanted to report it, it is valuabe to provide the :doc:`stack trace`
+ If there was an exception and you would like to report it, it is valuable to provide the :doc:`stack trace`
```

How about a different wording and fixed typo?